.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -70,3 +70,17 @@ This change added tests and can be verified as follows:
 
   - Does this pull request introduce a new feature? (yes / no)
   - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
+
+----
+
+# Review Progress
+
+**NOTE: THE REVIEW PROGRESS MUST ONLY BE UPDATED BY AN APACHE FLINK COMMITTER!**
+
+* [ ] 1. The contribution is well-described.
```

Comment:
Should we add a field to add the committers name to track who made this decision?

Reply:
Shouldn't this be tracked by the comments in Github and also posted to the mailing list?

Reply:
I'm sure it is tracked somewhere. But I thought it might make sense to put a name right next to certain decisions. And it helps if there is a committer that takes ownership.